waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -109,6 +110,24 @@ Press any key to continue or Ctrl+C to cancel.`);
 		replaceLineInLocalToml(minMachinesOptionRegex, 'min_machines_running = 1');
 	}
 
+	if (!doesLocalTomlContainLine(internalPortOptionRegex)) {
+		await question(`\n⚠️  There was an issue setting up your server app.
+We tried modifying your server fly.toml to set ${boldText(
+		'internal_port = 8080',
+	)}, but couldn't find the option ${boldText(
+	'internal_port',
+)} in the fly.toml.
+
+This means your server app might not be accessible.
+
+Contact the Wasp Team at our Discord server if you need help with this: https://discord.gg/rzdnErX
+
+Press any key to continue or Ctrl+C to cancel.`);
+	} else {
+		// the default fly.toml assumes port 8080 (or 3000, depending on flyctl version).
```

Comment:
I feel like this comment is redundant - we are setting the value to something we want, the default is not really important to know for the reader.

Reply:
In this file, here: https://github.com/wasp-lang/wasp/blob/a0f108b8f07b00b9b55e094e114148e1a3a78588/waspc/packages/deploy/src/providers/fly/setup/setup.ts#L139 we are also setting the server env var to `8080` and we want these two to change at the same time. So, I'd extract the port information into a variable and use it in both places e.g. `serverPort` or similar.